eng/targets/Helix.Common.props

```diff
@@ -27,9 +27,7 @@
       <!-- aspnetcore-ci[-official] (ci.yml) -->
       <!-- aspnetcore-quarantined-pr (quarantined-pr.yml) -->
       <ItemGroup>
-        <HelixAvailableTargetQueue Include="Ubuntu.2004.Amd64.Open" Platform="Linux" />
```

Comment:
Why get rid of this + Win?

Reply:
This is a draft PR intended _only_ to test whether everything runs smoothly on OSX 13. No need to run tests anywhere else.